driver/build/Dockerfile

```diff
@@ -20,8 +20,8 @@ ENV REVISION $commit
 RUN CGO_ENABLED=0 GOOS=$TARGETOS GOARCH=$TARGETARCH go build -a -ldflags "-X 'main.gitCommit=${REVISION}' -extldflags '-static'" -o _output/ibm-spectrum-scale-csi ./cmd/ibm-spectrum-scale-csi
 # RUN chmod +x,u+s _output/ibm-spectrum-scale-csi
 
-FROM registry.access.redhat.com/ubi9-minimal:9.4-1227.1726694542
-ARG version=2.13.0
+FROM registry.access.redhat.com/ubi9-minimal:9.5
```

Comment:
@Jainbrt @hemalathagajendran please note that I also updated the image tag so we start pulling `ubi9-minimal:9.5`.